source/routing/redirection.md

```diff
@@ -77,17 +83,22 @@ App.TopChartsChooseRoute = Ember.Route.extend({
 });
 
 // Superclass to be used by all of the filter routes below
-App.FilterRoute = Ember.Route.extend({
+// app/filter/route.js
+export default Ember.Route.extend({
   activate: function() {
     var controller = this.controllerFor('application');
     controller.set('lastFilter', this.templateName);
   }
 });
 
-App.TopChartsSongsRoute = App.FilterRoute.extend();
-App.TopChartsAlbumsRoute = App.FilterRoute.extend();
-App.TopChartsArtistsRoute = App.FilterRoute.extend();
-App.TopChartsPlaylistsRoute = App.FilterRoute.extend();
+// app/top-charts-songs/route.js
```

Comment:
I already fixed it, no worries. It _was_ a bit of a pain.I basically needed to hand audit/fix all the guides over Sunday, which kind of negates the help we got :-/